addons/point_of_sale/static/src/app/store/pos_store.js

```diff
@@ -1286,6 +1285,13 @@ export class PosStore extends Reactive {
         }
     }
 
+    get linesToRedund() {
+        return this.models["pos.order"].getAll().reduce((acc, order) => {
```

Comment:
Do you need the getAll here ? If yes, why not use it there
`const allLineToRefundUuids = this.models["pos.order"].reduce((acc, order) => {
            Object.assign(acc, order.uiState.lineToRefund);
            return acc;
        }, {});`
 if not, remove it